### PR TITLE
Greek translation

### PR DESCRIPTION
**Pull request by christolis**
- [x] Have you followed the [contributing guidelines](https://github.com/github/opensource.guide/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you explained what your changes do, and why they add value to the Guides?

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

-----
With this pull request, I am aiming to provide a space for collaborators who are able to assist with translating the project into the Greek language.

## TODO List
- [x]  el.yml
- [x]  index.html
- [x]  best-practices.md
- [x]  building-community.md
- [x]  code-of-conduct.md
- [x]  finding-users.md
- [x]  getting-paid.md
- [x]  how-to-contribute.md
- [x]  leadership-and-governance.md
- [x]  legal.md
- [x]  metrics.md
- [x]  starting-a-project.md

## Standard Greek Vocabulary Table
|:gb: English|:greece: Greek|
|-|-|
|maintainer|συντηρητής|
|evangelize|προωθώ|
|contributor|συνεισφέρων|
|project|πρότζεκτ|
|contribute|συνεισφέρω|
|open source|ανοιχτός κώδικας|

### CODE REVIEW

**Review by Mitumaru — COMMENTED** [review]
My NFT is token change

**Review by puchongcnxx29 — COMMENTED** [review]
thanks

**Review by Shahbaz9594 — COMMENTED** [review]
Approved

**Comment by Mitumaru** [comment]
> Approved

**Review by Mitumaru — COMMENTED** [review]
Okay I'm sorry

**Review by Errorzim — COMMENTED** [review]
I accept it’s

**Comment by Henrytech22** [comment]
Am good with whatever it is with BNB

**Review by Henrytech22 — COMMENTED** [review]
Good app, Bnb

**Review by MikeMcQuaid — CHANGES_REQUESTED** [review]
Looking good so far! Please re-request review when it's completed.

**Review by Thamerhaliq — COMMENTED** [review]
Salt

**Comment by ChristolisOfficial** [comment]
> Looking good so far! Please re-request review when it's completed.

Thank you! Will do. Easter holidays are almost over here in my place so I'll get some more progress done in the next few days! 🚀

File: _articles/el/how-to-contribute.md [hunk 1]
```diff
@@ -0,0 +1,528 @@
...
+description: Θέλετε να συνεισφέρετε σε πρότζεκτς ανοιχτού κώδικα; Ένας οδηγός για τη συνεισφορά στον ανοιχτό κώδικα, για αρχάριους και βετεράνους.
+class: contribute
+order: 1
+image: /assets/images/cards/contribute.png
+related:
+  - beginners
+  - building
+---
+
+## Why contribute to open source?
```
[the remark this answers is not in the document]
Comment (paul1974nsac) [reply _articles/el/how-to-contribute.md:13]: > H

[the remark this answers is not in the document]
Comment (paul1974nsac) [reply _articles/el/how-to-contribute.md:13]: paul1974nsac

[the remark this answers is not in the document]
Comment (paul1974nsac) [reply _articles/el/how-to-contribute.md:13]: paul1974nsac

File: _articles/el/how-to-contribute.md [hunk 2]
```diff
@@ -0,0 +1,528 @@
...
+  <p markdown="1" class="pquote-credit">
+— @errietta, ["Why I love contributing to open source software"](https://www.errietta.me/blog/open-source/)
+  </p>
+</aside>
+
+Contributing to open source can be a rewarding way to learn, teach, and build experience in just about any skill you can imagine.
+
+Why do people contribute to open source? Plenty of reasons!
+
+### Improve software you rely on
```
[the remark this answers is not in the document]
Comment (paul1974nsac) [reply _articles/el/how-to-contribute.md:27]: > Thamer

File: _articles/el/how-to-contribute.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (paul1974nsac) [reply _articles/el/how-to-contribute.md:13]: paul1974nsac

**Comment by BigsteppA1** [comment]
Can I review?

**Review by jjjjj29 — COMMENTED** [review]
Yy

**Comment by jiivii252** [comment]
Duit dimn duit ...

File: _articles/el/how-to-contribute.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Imadkhidre) [reply _articles/el/how-to-contribute.md:13]: [](url)16394374

**Comment by rmdanaasd** [comment]
ممكن حد يفهمني  ويسعدني  علي البرنامج

File: _articles/el/how-to-contribute.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Viileeu72) [reply _articles/el/how-to-contribute.md:13]: > 16394374

**Comment by Viileeu72** [comment]
> Can I review?

File: _articles/el/how-to-contribute.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Viileeu72) [reply _articles/el/how-to-contribute.md:13]: > 16394374

**Comment by GODWILLNFIRE** [comment]
i really am a begginer and someone has stole my life and identity im really
trying to fight them off  can u teach me

**Review by Nuttapon43 — COMMENTED** [review]
@christolis

**Comment by Jhstvf4t** [comment]
I don't have money anymore to go out to the workshop, I want to eat, let alone me

**Review by 1233duy — COMMENTED** [review]
Aa

> ``

**Comment by MikeMcQuaid** [comment]
Passing on this for now. @christolis please reopen when complete and passing CI, thanks!

**Comment by christolis** [comment]
@MikeMcQuaid The rest of the articles have now been properly translated into the Greek language! My apologies for the massive delay! For some reason, I am unable to reopen the pull request. Should I re-create it?

**Comment by MikeMcQuaid** [comment]
@christolis Reopened now! Please rebase and push to this branch.

**Review by freaki2 — COMMENTED** [review]
Approved

**Review by MikeMcQuaid — APPROVED** [review]
Thanks so much for your contribution! Without people like you submitting PRs we couldn't run this project. You rock, @christolis!

**Pull request merged**